### PR TITLE
Bug 1194996 - create a simple API to fetch build information for a build

### PR DESCRIPTION
**Pull request by parkouss**

### CODE REVIEW

File: mozregression/fetch_build_info.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+:class:`InboundInfoFetcher`. they both inherit from the :class:`InfoFetcher`.
+
+Here is an example of usage, to get the build url of builds::
+
+  from datetime import date
+  from mozregression.fetch_config import create_config
+
+  fetch_config = create_config('firefox', 'linux', 64)
+  # note that you should configure the fetch_config to use another repo
+  # than the defaut one for the application.
```
Comment (wlach) [inline mozregression/fetch_build_info.py]: I'm not sure what this comment means.

File: mozregression/fetch_build_info.py [hunk 2]
```diff
@@ -0,0 +1,230 @@
+"""
+This module offers an API to get information for one build.
+
+The public API is composed of two classes, :class:`NightlyInfoFetcher` and
+:class:`InboundInfoFetcher`. they both inherit from the :class:`InfoFetcher`.
+
+Here is an example of usage, to get the build url of builds::
```
Comment (wlach) [inline mozregression/fetch_build_info.py]: to get the build url of builds -> to get the urls for a set of builds

File: mozregression/fetch_build_info.py [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+            if matched:
+                data['changeset'] = matched.group(1)
+        return data
+
+    def find_build_info(self, changeset_or_date, fetch_txt_info=True):
+        """
+        Abstract method to retrieve build information over the internet for
+        one build.
+
+        This returns a dict that contain build information. At minima, this
```
Comment (wlach) [inline mozregression/fetch_build_info.py]: At minima -> At minimum

**Comment by parkouss** [comment]
Thanks @Will - I addressed the issues in a follow-up, i.e. https://bugzilla.mozilla.org/show_bug.cgi?id=1197128

**Comment by will** [comment]
you mean @wlach I think

**Pull request merged**